sources/packages/backend/libs/integrations/src/services/disbursement-schedule/e-cert-processing-steps/validate-disbursement-part-time-step.ts

```diff
@@ -38,6 +47,63 @@ export class ValidateDisbursementPartTimeStep
       );
       shouldContinue = false;
     }
+    this.validateCSLPLifetimeMaximumCheckStep(
+      eCertDisbursement,
+      entityManager,
+      log,
+    );
     return shouldContinue;
   }
+
+  /**
+   * Validate if CSLP disbursed exceeded the lifetime maximums.
+   * @param eCertDisbursement eligible disbursement to be potentially added to an e-Cert.
+   * @param entityManager used to execute the commands in the same transaction.
+   * @param log cumulative log summary.
+   */
+  private async validateCSLPLifetimeMaximumCheckStep(
```

Comment:
Please remove the step from the method name to avoid confusion with the other steps. IMO it is not adding too much value to the name.